.github/workflows/publish-image.yml

```diff
@@ -39,9 +43,10 @@ jobs:
           labels: |
             org.cyrusimap.cyrus-docker.built-from=${{ github.sha }}
           tags: |
-            type=schedule
-            type=ref,event=branch
-            type=raw,value=bookworm,enable=${{ github.ref == format('refs/heads/{0}', 'master') }}
+            type=schedule,enable=${{ matrix.platform != '' }}
```

Comment:
This changes to nightly tagging an arm build. I don't think we want that, we want the amd64 build instead

Reply:
Fixed simultaneous to you asking!